modules/tinymce/src/themes/silver/main/ts/ui/menus/item/structure/ItemStructure.ts

```diff
@@ -83,11 +83,13 @@ const renderColorStructure = (item: ItemStructureSpec, providerBackstage: UiFact
 };
 
 const renderItemDomStructure = (ariaLabel: Optional<string>): RawDomSchema => {
-  const domTitle = ariaLabel.map((label): { attributes?: { title: string }} => ({
+  const domTitle = ariaLabel.map((label): { attributes?: { title: string, id?: string }} => ({
     attributes: {
       // TODO: AP-213 change this temporary solution to use tooltips, ensure its aria readable still.
       // for icon only implementations we need either a title or aria label to satisfy aria requirements.
-      title: I18n.translate(label)
+      title: I18n.translate(label),
+      // TODO: No idea if this is good
+      id: Id.generate('menu_item')
```

Comment:
Maybe using Alloy `uid` as element id is better than generating one

Reply:
I would prefer to use uid, yes. I think we already use it for other aria link purposes 🤔

Reply:
Actually, aria link uses `Id.generate`. Also, don't think it is possible to use `uid` here since it is generated in the gui build process.